### ISSUE TITLE
AudioEncoderConfig.bitrate should be explicitly forbidden to be 0.

### ISSUE THREAD

**Issue by jyavenard** [w3c/webcodecs#860]
https://w3c.github.io/webcodecs/#dom-audioencoderconfig-bitrate doesn't define any constraints for the bitrate.

But we currently test that the bitrate is not 0 in 
w3c/web-platform-tests/webcodecs/audio-encoder-config.https.any.js `'Bit rate present but equal to zero'`

**Comment by Djuffin** [comment]
I agree, if `sampleRate`, `numberOfChannels` or `bitrate` are zero, we should declare the config invalid.
